jenkinsapi/fingerprint.py

```diff
@@ -71,6 +71,7 @@ def valid(self) -> bool:
         return True
 
     def validate_for_build(self, filename: str, job: str, build: int) -> bool:
+        _ = filename  # Currently unused
```

Comment:
While the `filename` argument to `validate_for_build()` is now unused I did not want to change the API by removing it from the function signature.  The assignment to `_` pacifies pylint so that it does not emit an "unused" message.